### ISSUE TITLE
Model Bringup: Vilt

### ISSUE THREAD

**Issue by kamalrajkannan78** [tenstorrent/tt-forge-fe#484]
## Requirements

| Framework | Input source | Task               | variant                                              | Device Type | Completed | Blocker |
|-----------|--------------|--------------------|------------------------------------------------------|-------------|-----------|---------|
| pytorch   | Transformers | Question answering | vilt_question_answering_hf_pytorch_b32-finetuned-vqa | N150        | ❌         |         |
| pytorch   | Transformers | Masked LM          | vilt_maskedlm_hf_pytorch_vilt-b32-mlm                | N150        | ❌         |         |
